### DISCUSSION TITLE
A few bugs I've noticed + Feature Requests

### DISCUSSION THREAD

**Discussion by jdev082** [AppOutlet/AppOutlet#276] [General]
- Icons load very slow, takes a few minutes.
- Only app images show up - #227 

Feature Requests
- Auto run with necessary arguments on Nvidia graphics.
- Native system package support.
- Repository management + Updates

**Comment by MessiasLima** [comment]
Hello @JaydenDev 

> Icons load very slow, takes a few minutes.

It is not related to the App Outlet directly. You are talking about AppImage icons, aren't you? https://github.com/AppImage/appimage.github.io/issues/2800 

> Auto run with necessary arguments on Nvidia graphics

It is an electron issue. Please see #264 and #274

> Native system package support.

I'm already working on it. Please see https://github.com/app-outlet/node-packagekit . You are welcome to help me if you want

**Comment by MessiasLima** [comment]
> Repository management + Updates


 Could you please elaborate on this?

**Comment by MessiasLima** [comment]
> Only app images show up

It is already tracked.
